common/config.py

```diff
@@ -494,6 +494,12 @@ def setSnapshotsPath(self, value, profile_id = None, mode = None):
 
             return False
 
+        elif fs.startswith('ntfs'):
+            self.notifyError(
+                _('Destination filesystem for {path} is formatted with NTFS '
+                  'which has known incompatibilities with Unix style file systems.')
```

Comment:
I am not skilled with English words. I just asked the "stochastic parrot" and modified the string a bit. So feel free to modify it either.

```suggestion
                _('The destination filesystem for {path} is formatted with NTFS, '
                  'which has known incompatibilities with Unix-style filesystems.')
```

Reply:
I'm no english major either (GIBBeriSh is my family language :) ) ... I was just matching the existing message pattern in config.py.